tests/test_hep_bdFFT.py

```diff
@@ -108,7 +172,57 @@ def test_documents_from_FFT_ignores_context():
 
     result = hep.do(create_record(snippet))
 
-    assert '_fft' not in result
+    assert 'documents' not in result
+    assert 'figures' not in result
+
+
+def test_documents_from_FFT_does_not_require_s():
+    schema = load_schema('hep')
+    subschema = schema['properties']['documents']
+
+    snippet = (
+        '<datafield tag="FFT" ind1=" " ind2=" ">'
+        '  <subfield code="a">http://www.mdpi.com/2218-1997/3/1/24/pdf</subfield>'
+        '  <subfield code="d">Fulltext</subfield>'
+        '  <subfield code="t">INSPIRE-PUBLIC</subfield>'
+        '</datafield>'
+    )  # DESY harvest
+
+    expected = [
+        {
+            'description': 'Fulltext',
+            'url': 'http://www.mdpi.com/2218-1997/3/1/24/pdf'
+        }
+    ]
+    result = hep.do(create_record(snippet))
+
+    assert validate(result['documents'], subschema) is None
+    expected[0]['key'] = result['documents'][0]['key']
+    assert expected == result['documents']
+
+    expected = [
+        {
+            'a': 'http://www.mdpi.com/2218-1997/3/1/24/pdf',
+            'd': 'Fulltext',
+            't': 'INSPIRE-PUBLIC',
+        },
+    ]
+    result = hep2marc.do(result)
+
+    assert expected == result['FFT']
+
+
+def test_fft_from_FFT_percent_percent():
+    snippet = (
+        '<datafield tag="FFT" ind1="%" ind2="%">'
+        '  <subfield code="a">/opt/cds-invenio/var/tmp-shared/apsharvest_unzip_5dGfY5/articlebag-10-1103-PhysRevD-87-083514-apsxml/data/PhysRevD.87.083514/fulltext.xml</subfield>'
+        '  <subfield code="o">HIDDEN</subfield>'
+        '  <subfield code="t">APS</subfield>'
+        '</datafield>'
+    )  # record/1094156
+
+    assert 'documents' not in hep.do(create_record(snippet))
```

Comment:
if you define `result = hep.do(create_record(snippet))`, you can reuse it instead of computing it twice.